core/chains/evm/txmgr/mocks/utils.go

```diff
@@ -7,18 +7,19 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 
 	"github.com/smartcontractkit/chainlink/v2/common/txmgr/types/mocks"
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/gas"
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/txmgr"
 	evmtypes "github.com/smartcontractkit/chainlink/v2/core/chains/evm/types"
 )
 
-type MockEvmTxManager = TxManager[*big.Int, *evmtypes.Head, common.Address, common.Hash, common.Hash]
+type MockEvmTxManager = TxManager[*big.Int, *evmtypes.Head, common.Address, common.Hash, common.Hash, *evmtypes.Receipt, gas.EvmFee, txmgr.NullableEIP2930AccessList]
```

Comment:
Yikes. Any thoughts on renaming `NullableEIP2930AccessList`?
```suggestion
type MockEvmTxManager = TxManager[*big.Int, *evmtypes.Head, common.Address, common.Hash, common.Hash, *evmtypes.Receipt, gas.EvmFee, txmgr.NullableEIP2930AccessList]
```
Or eliminating it? Why is this wrapper even necessary? Can we just use the basic type?
```go
// AccessList is an EIP-2930 access list.
type AccessList []AccessTuple
```
Maybe we were avoiding making nil v.s. empty slice significant? If so, think a `*AccessList` might be more clear.

Reply:
I think changing it to `*AccessList` long term achieves the same goal as having the `IsValid` bool - but i think that change potentially involves a DB change in serialization

how do you feel about either renaming it or aliasing it to something short like `EvmAccessList`?

Reply:
Ah OK. Yeah an alias sounds good. Either way.